yapapi/engine.py

```diff
@@ -125,6 +127,9 @@ def __init__(
         self._wrapped_consumers: List[AsyncWrapper] = []
 
         self._strategy = strategy
+        # set default property value ranges if they were not set in the market strategy
+        strategy.set_prop_value_ranges_defaults(DEFAULT_PROPERTY_VALUE_RANGES)
```

Comment:
I think we decided (or at least considered, and didn't decide against it?) to move this logic to `yapapi.strategy.MarketStrategy`, because it seems to belong there better.

Reply:
Done